source/users-groups-permissions.rst

```diff
@@ -46,27 +46,24 @@ You... ish.
     Users can be people or non-people, but as far as the OS is concerned both
     are almost identical.
 
-::
+.. code-block:: console
 
     $ whoami    # your username
     $ who       # who is logged in?
     $ w         # who is here and what are they doing?
     $ id        # user ID, group ID, and groups you're in
 
-Sometimes robots are users too: Apache, Mailman, ntp.
-
+Not just people: Apache, Mailman, ntp aka "system users"
```

Comment:
'ntp aka "system users"' is somewhat of a vague way to end this, they almost seem unrelated to the first part of the list. More separation between the two will help understanding.

like this:
Not just people: Apache, Mailman, ntp. "system users"